packages/flutter_tools/test/general.shard/run_hot_test.dart

```diff
@@ -29,6 +36,70 @@ void main() {
       const NoOpAnalytics(),
     );
   });
+
+  group('signal handling', () {
+    late final _FakeHotCompatibleFlutterDevice flutterDevice;
+
+    setUpAll(() {
+      flutterDevice = _FakeHotCompatibleFlutterDevice(FakeDevice());
+    });
+
+    late MemoryFileSystem fileSystem;
+
+    setUp(() {
+      fileSystem = MemoryFileSystem.test();
+    });
+
+    testUsingContext(
+      'kills the test device',
+      () async {
+        final HotRunner runner = HotRunner(
+          <FlutterDevice>[
+            flutterDevice,
+          ],
+          target: 'main.dart',
+          debuggingOptions: DebuggingOptions.disabled(BuildInfo.debug),
+          analytics: _FakeAnalytics(),
+        );
+
+        await runner.run();
+        await runner.cleanupAfterSignal();
+        expect(flutterDevice.wasExited, true);
```

Comment:
```suggestion
        expect(flutterDevice.wasExited, isTrue);
```

Reply:
(Here and below) My 2c: I don't think there is any value in this suggestion and we should stop making the suggestion - even the pkg/test authors are debating deprecating `isTrue` and `isFalse` because it has no particular value.

If you feel strongly about it, I can make the change anyway.

Reply:
SGTM. I just checked the output of these matchers when they fail and it's basically the same as not using the matcher.

> even the pkg/test authors are debating deprecating `isTrue` and `isFalse` because it has no particular value

This is news to me.